pkg/apiserver/registry/generic/key.go

```diff
@@ -32,6 +32,11 @@ func ParseKey(raw string) (*Key, error) {
 	for i := 0; i < len(parts); i += 2 {
 		k := parts[i]
 		if i+1 >= len(parts) {
+			// Kube aggregator just appends the name to a key
```

Comment:
@toddtreece -- I hit a wall trying to get the aggregator setup to have the correct prefix.  This does not seem terrible, and avoids needing too much custom setup

Reply:
👍 this seems fine for now